packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -59,90 +58,101 @@ Use the following steps to display the user consent dialog box as part of an Ope
     For the [Authorization Code flow](/docs/concepts/oauth-openid/#authorization-code-flow), the response type is `code`. You can exchange an authorization code for an ID token and/or an access token using the `/token` endpoint.
 
 1. Click **Save**.
-1. To enable consent for the [scopes](/docs/reference/api/authorization-servers/#create-a-scope) that you want to require consent for, select **Security** and then **API**.
-1. On the **Authorization Servers** tab, select **default** (Custom Authorization Server) in the table. In this example, we are enabling consent for default Custom Authorization Server scopes.
+1. To enable consent for [scopes](/docs/reference/api/authorization-servers/#create-a-scope), from the left navigation select **Security** and then **API**.
+1. On the **Authorization Servers** tab, select **default** (custom authorization server) in the table. In this example, we are enabling consent for a default custom authorization server scope.
 1. Select the **Scopes** tab.
-1. Click the edit icon for the **phone** scope. The Edit Scope dialog box appears.
+1. Click the edit icon for the **phone** scope. The Edit Scope dialog appears.
+1. For this use case example, select **Required** in the **User consent** section. **Required** indicates that the user must grant the app access to the information (scope) or they can't sign in to the app. The other options available include:
+
+    * **Implicit**: The default setting. Indicates that the user doesn't have to grant the app access to the information. User consent is implied.
+    * **Optional**: Indicates that the user can skip granting the app access to the information (scope). Scopes that the user skips aren't included in the authorization response. After a user skips a scope, the next time that they sign in, Okta doesn't prompt them for it. If you later make the scope required for the app, the user is then prompted to grant the app access to that scope.
 
-1. Select **Require user consent for this scope**. The **Block services from requesting this scope** check box is automatically selected.
+    > **Note:** When you include `prompt=consent` in the authorization request, the user is prompted for all consent-enabled scopes. This includes scopes that are required or optional, even when the user has already given consent or skipped a scope. Additionally, when previously required scopes appear in the dialog, the user has the option to then skip those scopes.
```

Comment:
By `previously required scopes` do you mean they were previously required, but now made optional. I'm not sure if that text conveys the information with clarity.
You can say `previously required scopes which are made optional`